Source/ASButtonNode+Private.h

```diff
@@ -31,14 +31,15 @@
   UIImage *_disabledBackgroundImage;
   
   CGFloat _contentSpacing;
-  BOOL  _laysOutHorizontally;
-  ASVerticalAlignment _contentVerticalAlignment;
-  ASHorizontalAlignment _contentHorizontalAlignment;
   UIEdgeInsets _contentEdgeInsets;
-  ASButtonNodeImageAlignment _imageAlignment;
   ASTextNode *_titleNode;
   ASImageNode *_imageNode;
   ASImageNode *_backgroundImageNode;
+
+    BOOL  _laysOutHorizontally;
```

Comment:
This change is genuinely awesome, and I'm so glad to see you in the community @bolsinga !!

Can you remove 2x indentation spaces here? hehe.